TestFoundation/TestCalendar.swift

```diff
@@ -111,6 +111,31 @@ class TestCalendar: XCTestCase {
         XCTAssertEqual(components.day, 18)
 
     }
+    
+    func test_gettingDatesOnJapaneseCalendar() throws {
```

Comment:
You'll need to add this test into the list of tests at the end of the file, Linux and TestFoundation doesnt automatically pick up tests as happens on macOS. If you can squash that change in we can get this merged, thanks

Reply:
@spevans Sure. Fixed it. Now, It seems be okay.